pkg/config/containers.conf

```diff
@@ -351,6 +351,9 @@ default_sysctls = [
 #
 #env = []
 
+# Define where events events log will be stored, when events_logger is "file".
```

Comment:
```suggestion
# Define where event logs will be stored when events_logger is "file".
```

Reply:
These are events about the events log?

Fixed in both places.